### PR TITLE
Add TokenEndPoint and AuthorizationEndpoint explanation

### PR DESCRIPTION
**Pull request by erikjanwestendorp**
Is it a good idea to explain that the TokenEndpoint and AuthorizationEndpoint must also be configured if it is a Single Tenant configuration?

### CODE REVIEW

**Review by nikolajlauridsen — APPROVED** [review]
Looks good to me 👍

File: Reference/Security/Authenticate-with-Active-Directory/index.md [hunk 1]
```diff
@@ -60,7 +62,7 @@ You need to install the `Microsoft.AspNetCore.Authentication.MicrosoftAccount` N
     ```
 
     :::note
-    Ensure to replace **{your_client_id}** and **{your_client_secret}** in the code with the values from the Azure AD tenant.
+    Ensure to replace **{your_client_id}** and **{your_client_secret}** in the code with the values from the Azure AD tenant. If Azure AD is configured to use only accounts in the organizational directory only (single tenant), you have to specify the Token and AuthorizationEndpoint as well.
```
Comment (sofietoft) [inline Reference/Security/Authenticate-with-Active-Directory/index.md:65]:
```suggestion
    Ensure to replace **{your_client_id}** and **{your_client_secret}** in the code with the values from the Azure AD tenant. If Azure AD is configured to use accounts in the organizational directory only (single tenant), you also have to specify the Token and AuthorizationEndpoint.
```

**Review by sofietoft — COMMENTED** [review]
Looks like we're all in the clear here @erikjanwestendorp ! 😄 
Thanks for the review @nikolajlauridsen 💪 

I just made a tiny suggestion to a sentence - to shorten it a bit 😅

**Comment by erikjanwestendorp** [comment]
@sofietoft and @nikolajlauridsen Thanks for the review. Just updated the PR with the suggestion. 😄

**Comment by sofietoft** [comment]
Amazing! Let's get this one merged in then 😄

**Pull request merged**